README.adoc

```diff
@@ -141,3 +141,24 @@ repoURL=<yours_repository>
 branch=<yours_branch>
 sha1=<your_last_commit_id_at_branch> 
 version=<the_James_version_of_this_branch>
+
+=== Tools
+
+==== Prometheus
```

Comment:
could we have a docker compose please?

Reply:
I'm scared I will have to backport everything from Jenkins 2...

Those tools are not required, that's why I only added this short documentation.